### PR TITLE
use latest sketches-core-0.10.1 and memory-0.10.3

### PR DESCRIPTION
**Pull request by AlexanderSaydakov**

### CODE REVIEW

**Comment by drcrallen** [comment]
Thank you @AlexanderSaydakov , are there specific features or highlights that can be called out? or is this just a keeping-up-with-master kind of patch?

**Comment by niketh** [comment]
👍

**Comment by AlexanderSaydakov** [comment]
@drcrallen This should be fully compatible. Mostly maintenance and preparation to submit aggregators for other sketch types.

File: extensions-core/datasketches/pom.xml [hunk 1]
```diff
@@ -38,7 +38,12 @@
     <dependency>
       <groupId>com.yahoo.datasketches</groupId>
       <artifactId>sketches-core</artifactId>
-      <version>0.8.4</version>
+      <version>0.10.1</version>
+    </dependency>
+    <dependency>
+      <groupId>com.yahoo.datasketches</groupId>
+      <artifactId>memory</artifactId>
```
Comment (himanshug) [inline extensions-core/datasketches/pom.xml]: why do we explicitly need to define memory dependency, would't it come based on sketches-core version:?

**Comment by akashdw** [comment]
👍

**Pull request merged**